man/IRkernel.Rd

```diff
@@ -9,9 +9,7 @@
 \title{An R kernel for Jupyter.}
 \format{An object of class \code{list} of length 5.}
 \usage{
-options(jupyter.* = ...)
-getOption('jupyter.*')
-jupyter_option_defaults$jupyter.*
+jupyter_option_defaults
```

Comment:
Looking at the manual on cran, I think we should rework out docstrings anyway... It's a bit sad that in repr none of the actual `repr_html`, `repr_latex`... are actually in the table of content of the PDF :-(